build.gradle.kts

```diff
@@ -62,20 +62,23 @@ tasks {
         group = "doc"
         args.set(listOf("--mutex", "network"))
         workingDir.set(file("${rootDir}/documentation"))
+        environment.set(mapOf("NODE_OPTIONS" to "--openssl-legacy-provider"))
```

Comment:
After upgrade, Node.js 24 uses OpenSSL 3.0 which doesn't support the legacy MD4 hash algorithm that the old webpack in Docusaurus 2.0.0-beta.8 relies on. Adding env NODE_OPTIONS to use legacy OpenSSL provider.